packages/mui-joy/src/Box/BoxProps.ts

```diff
@@ -1,10 +1,28 @@
 import { OverrideProps } from '@mui/types';
 import { BoxTypeMap } from '@mui/system';
 import { Theme } from '../styles/types';
+import { SlotProps, CreateSlotsAndSlotProps } from '../utils/types';
 
 export type BoxSlot = 'root';
 
+export interface BoxSlots {
+  /**
+   * The component that renders the root.
+   * @default 'div'
+   */
+  root: React.ElementType;
+}
+
+export type BoxSlotsAndSlotProps = CreateSlotsAndSlotProps<
+  BoxSlots,
+  {
+    root: SlotProps<'div', {}, BoxOwnerState>;
+  }
+>;
+
```

Comment:
Please revert this, `Box` does not have initial styles so `slots` prop does not make sense here.